app/src/sharedTest/java/org/oppia/app/administratorcontrols/AppVersionActivityTest.kt

```diff
@@ -78,7 +79,7 @@ class AppVersionActivityTest {
         withText(
           String.format(
             context.resources.getString(R.string.app_version_name),
-            context.packageManager.getPackageInfo(context.packageName, 0).versionName
+            context.packageManager.getVersionName()
```

Comment:
Is this used elsewhere in the codebase? If so, suggest moving the extension function to a Kotlin helper file in the utility package and importing it everywhere it's needed.

Reply:
Done!

Reply:
I also implemented an extension function for `lastUpdateTime` because I saw it was needed just as much as `versionName`.